.circleci/config.yml

```diff
@@ -82,6 +82,7 @@ jobs:
             cmake -DCMAKE_BUILD_TYPE=Release -DWITH_GUDHI_EXAMPLE=OFF -DWITH_GUDHI_UTILITIES=OFF -DWITH_GUDHI_PYTHON=ON -DPython_ADDITIONAL_VERSIONS=3 -DWITH_GUDHI_REMOTE_TEST=ON .
             cd python
             python3 setup.py build_ext --inplace
+            ctest --output-on-failure
             make sphinx
```

Comment:
How do those 2 lines interact? I usually build the sphinx doc first, and ctest runs the sphinx tests. Does it still manage to run the sphinx tests before the build? Or does it build sphinx and make the next line redundant?

Reply:
`sphinx_py_test` is independant to `make sphinx`, and `make sphinx` do not launch autodoc tests.
No it is not redundant.